### PR TITLE
New Disease that turns people into tiny lizards

### PR DESCRIPTION
**Pull request by Battletrap360**
:cl: battletrap360
add: Added Heat-Lamp Syndrome, a disease that turns the infected into a tiny, harmless lizard.
/:cl:

I am adding this because It is the first project I have done coding SS13, and my efforts produced a simple, yet fun disease that could be of use to Admemes.

### CODE REVIEW

**Comment by ike709** [comment]
>adminordrazine can't cure it

**Comment by SmokingKilz** [comment]
What cures it? If nothing, something should cure it. Otherwise, why not.

**Comment by Battletrap360** [comment]
The Disease happens fairly quickly, and would only be given to someone by an admin, so I think a cure isn't really needed.

**Comment by ike709** [comment]
yet it still has the `CURABLE` flag

**Comment by Battletrap360** [comment]
Where?

**Comment by ike709** [comment]
it is inherited from the `/datum/disease/transformation` parent

**Comment by Battletrap360** [comment]
Welp, do I edit the code and make a new request, or is there some way to edit it here?

**Comment by ike709** [comment]
you can add additional commits to a PR by doing changes on the same branch

**Comment by Battletrap360** [comment]
gotcha

**Comment by Battletrap360** [comment]
Figured it was easier to just add a cure, so here it is.

**Comment by Battletrap360** [comment]
Should have gave that commit a better name, but It gets the job done.

**Comment by ike709** [comment]
commit names are irrelevant

**Comment by Battletrap360** [comment]
alrighty

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -239,3 +239,21 @@
 	stage4	= list("<span class='danger'>You're ravenous.</span>")
 	stage5	= list("<span class='danger'>You have become a morph.</span>")
 	new_form = /mob/living/simple_animal/hostile/morph
+
+/datum/disease/transformation/lizard
+	name = "Heat-Lamp Syndrome"
+	max_stages = 5
+	cure_text = "Unknown Cure"
+	cures = list("adminordrazine")
+	agent = "LizardTears"
```
Comment (FlattestGuitar) [inline code/datums/diseases/transformation.dm]: add a space here, we do not encourage poor grammar

**Pull request merged**